### ISSUE TITLE
Add option to 'mark as read' when marking an email or bundle as 'done'

### ISSUE THREAD

**Issue by tswitkowski** [teresa-ou/inboxy#56]
This does not replicate what Inbox did, but replicates what I do in gmail.
The benefit is that the labels don't show in the label list, if you configure the label to to only show when there are unread mails.

**Comment by teresa-ou** [comment]
I can see how this might be useful, although it seems like a pretty specific use case / workflow. If there's more interest from other users in this, then I'll consider adding this option.

**Comment by jbharrell** [comment]
I would use a 'mark as read' button or a 'mark as read and archive' button or both :) I may even pull the source and give a go at adding at least a 'mark as read' button to start.

**Comment by durandom** [comment]
@teresa-ou I would rework the "double-checkmark" button to just select all mails in the bundle. Then the user can call any action on the selected mails.
Alternatively add a keyboard shortcut to select all mails in the current bundle

**Comment by 71** [comment]
I agree. Being able to select mails in a bundle would be nice because, depending on the bundle, I won't process mails the same way (I will delete some bundles and archive others).

IMO the best would be to provide all four buttons (Select, Archive, Delete, View All) and to allow the user to toggle them on and off in the extension settings. Looking at the code, adding the two other buttons seems trivial; the hard part is managing user preferences.

**Comment by eflynch** [comment]
There may be overlap in usecase with this feature request I filed: https://github.com/teresa-ou/inboxy/issues/94

In particular, the "readness" of my archived emails is something I would prefer to preserve and would rather be able to retool the gmail interface to account for the fact that "readness" isn't the primary thing I care about compared to "doneness" wherever it comes up (e.g. in the label list on the left)

**Comment by benoror** [comment]
Hi! Optional "mark as read when archiving a bundle" is not built yet in **Inbundly**.

Workaround: use the **select-all** checkbox on the bundle, then Gmail's Mark as read (and Archive if you want both). If you want a single control that does both, please open a feature request (same family as #74 / #87).

Chrome: https://chromewebstore.google.com/detail/inbundly/pbfjicjmcpogjlbpljebhhgkgfhbdcga
Issues: https://github.com/benoror/inbundly/issues/new
